jenkins/ci.suse.de/templates/cloud-socok8s-integration.yaml

```diff
@@ -5,23 +5,6 @@
     number-to-keep: 30
     days-to-keep: 30
     script-path: Jenkinsfile.integration
-    parameters:
-      - choice:
-          name: deployment
-          choices:
-            - 'airship'
-            - 'osh'
-          default: 'airship'
```

Comment:
We don't have default values for these parameters in `Jenkinsfile.integration`. I'm afraid if you remove these it would be even worse.

Although we also have `ses_version` (https://github.com/SUSE-Cloud/socok8s/blob/master/Jenkinsfile.integration#L27) without default. Could it be this the offender?

Reply:
yes we do. 

deployment: https://github.com/SUSE-Cloud/socok8s/blob/master/Jenkinsfile.integration#L25
tempest_test_ype: https://github.com/SUSE-Cloud/socok8s/blob/master/Jenkinsfile.integration#L26

Reply:
I could be the ses_version, but Im not sure how this even works. Why do we need to set those params in both places? We used to have no params in this file and let the params go to default, so that must not be it.

I removed some leftover params from the other job, it may be that?

Reply:
for example, caasp4 params were in there for a long time while not reflected here, and the jobs were ok, so Im not sure we need that. This definition and the jenkinsfile seems to be separated, like

Reply:
Those are implicit defaults (first value in array), not documented (https://jenkins.io/doc/book/pipeline/syntax/#parameters). But probably not the guilty ones here.